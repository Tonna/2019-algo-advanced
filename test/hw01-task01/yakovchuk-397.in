25
1 1 1 3 1 5 2 2 3 6 6 1 5 5 13 15 16 11 13 1 10 14 22 20
6
25 6
1 5
15 8
22 1
24 6
18 4
