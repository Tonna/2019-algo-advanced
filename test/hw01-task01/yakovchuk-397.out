39
7
20
10
21
31
